### PR TITLE
Update content-caching.adoc

### PR DESCRIPTION
**Pull request by johannabecker**

### CODE REVIEW

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-
-== Preliminary information about content caching
-
-Content caching is only available from plentyShop LTS and IO versions 2.12 onwards. Download the current versions of the plugins on the plentymarkets https://marketplace.plentymarkets.com/plugins/templates[marketplace] or update the plugins in the *Plugins » Plugin overview*.
+This page serves to give you an overview about the new content caching functionality of https://github.com/plentymarkets/plugin-ceres[plentyShop LTS] and https://github.com/plentymarkets/plugin-io[IO]. The caching of content data of the online store prevents a recalculating of pages upon each new page view and therefore improves the performance of the online store. Data which has already been loaded or generated once remains in the cache and can be invoked much faster when it is accessed again.
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'therefore' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'been loaded' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'be invoked' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'it is' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: 🚫 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Microsoft.Contractions] Use 'it's' instead of 'it is'.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:3]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is accessed' may be passive voice. Use active voice if you can.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
 
 == Information about caching and invalidation
 
-* The cached data is retained for 7 days unless invalidated.
 * All online store pages are cached in IO *with the exception of user-dependent data*, which is loaded via REST. The following pages are not written into the cache:
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:7]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'are cached' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:7]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is loaded' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:7]: 🚫 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Microsoft.Contractions] Use 'aren't' instead of 'are not'.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 3]
```diff
@@ -22,10 +17,10 @@ Content caching is only available from plentyShop LTS and IO versions 2.12 onwar
 ** Saving and publishing a plugin set
 ** Changing the settings of a plugin and saving the new settings
 ** Linking layout containers in the settings of a plugin
-* The cache for individual variation pages is invalidated when changes are made to the item settings of the variation. The invalidation of variation pages is dependent on the refreshing of the Elasticsearch index, meaning that the aforementioned invalidation cycle of 5 minutes does not apply; the Elasticsearch index is refreshed every  10 to 15 minutes.
+* The cache for individual variation pages is invalidated when changes are made to the item settings of the variation. The invalidation of variation pages is dependent on the refreshing of the Elasticsearch index, meaning that the aforementioned invalidation cycle of 5 minutes does not apply; the Elasticsearch index is refreshed every 10 to 15 minutes.
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:20]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is invalidated' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:20]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'are made' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:20]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'aforementioned' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:20]: 🚫 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Microsoft.Contractions] Use 'doesn't' instead of 'does not'.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:20]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is refreshed' may be passive voice. Use active voice if you can.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 4]
```diff
@@ -22,10 +17,10 @@ Content caching is only available from plentyShop LTS and IO versions 2.12 onwar
...
 
-The link:https://knowledge.plentymarkets.com/en/basics/first-steps/plugins#60[Preview mode] in the *Plugins » Plugin overview* menu completely circumvents the content caching. Changes to the online store will therefore be displayed
-immediately in the preview mode.
+The link:https://knowledge.plentymarkets.com/en/basics/first-steps/plugins#60[Preview mode] in the *Plugins » Plugin overview* menu completely circumvents the content caching. Changes to the online store will therefore be displayed immediately in the preview mode. +
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:22]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Weasel] 'completely' is a weasel word!

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:22]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'therefore' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:22]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'be displayed' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:22]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Weasel] 'immediately' is a weasel word!

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 5]
```diff
@@ -22,10 +17,10 @@ Content caching is only available from plentyShop LTS and IO versions 2.12 onwar
...
+You can also add a query parameter like `?=nocache` to the URL to not load the page from the cache.
 
 == How to cache additional pages
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:25]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'additional' is too wordy.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 6]
```diff
@@ -43,7 +38,7 @@ $cacheRepository = pluginApp(ContentCacheRepositoryContract::class);
 $cacheRepository->enableCacheForResponse();
 
 ...
-  
+
 ----
 
 By integrating this code, you ensure that the additional routes are written into the cache. You can see how the code is implemented in the LayoutController of *IO*, located in the IO/src/Controllers/LayoutController.php file.
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:44]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'additional' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:44]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'are written' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:44]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is implemented' may be passive voice. Use active voice if you can.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 7]
```diff
@@ -62,7 +57,7 @@ $contentCacheRepo = pluginApp(ContentCacheRepositoryContract::class);
 $contentCacheRepo->linkVariationsToResponse($variationIds); // $variationIds = [1,2,3];
 
 ...
-  
+
 ----
 
 In this case, the page is automatically invalidated whenever changes are being made to the variations with the IDs 1, 2 or 3.
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:63]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'being made' may be passive voice. Use active voice if you can.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 8]
```diff
@@ -101,14 +96,14 @@ You can then implement the `registerIncluded()` method at any point in the **boo
 The example below shows how the method is implemented with an array of two additional query parameters, "springEvent" and "returnCustomer":
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:96]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is implemented' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:96]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'additional' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:96]: 🚫 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Vale.Spelling] Did you really mean 'springEvent'?

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:96]: 🚫 **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Vale.Spelling] Did you really mean 'returnCustomer'?

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 9]
```diff
@@ -101,14 +96,14 @@ You can then implement the `registerIncluded()` method at any point in the **boo
...
 $contentCacheQueryParamsRepository = pluginApp(ContentCacheQueryParamsRepositoryContract::class);
 $contentCacheQueryParamsRepository->registerIncluded([
     'springEvent',
     'returnCustomer'
 ]);
-    
+
 ----
 
 If a page that includes either one of these query parameters is accessed, a new cache entry is created on S3. This new entry differs from the basic entry, which is the cache entry for pages without additional query parameters. Therefore, both the page with and without the registered query parameters can be loaded from the ShopBooster cache.
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:109]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is accessed' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:109]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is created' may be passive voice. Use active voice if you can.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:109]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'additional' is too wordy.

**Review by plenty-cicd-3 — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>vale</summary>

[docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc|109 col 228|](http://github.com/plentymarkets/plenty-developers-docs/blob/48e144794e45fe555b6447b2a71628ee383c9fff/docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc#L109) [write-good.TooWordy] 'Therefore' is too wordy.
[docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc|109 col 306|](http://github.com/plentymarkets/plenty-developers-docs/blob/48e144794e45fe555b6447b2a71628ee383c9fff/docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc#L109) [write-good.Passive] 'be loaded' may be passive voice. Use active voice if you can.
[docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc|119 col 14|](http://github.com/plentymarkets/plenty-developers-docs/blob/48e144794e45fe555b6447b2a71628ee383c9fff/docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc#L119) [write-good.TooWordy] 'implement' is too wordy.
[docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc|119 col 241|](http://github.com/plentymarkets/plenty-developers-docs/blob/48e144794e45fe555b6447b2a71628ee383c9fff/docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc#L119) [write-good.Weasel] 'only' is a weasel word!
[docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc|119 col 275|](http://github.com/plentymarkets/plenty-developers-docs/blob/48e144794e45fe555b6447b2a71628ee383c9fff/docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc#L119) [write-good.Passive] 'is excluded' may be passive voice. Use active voice if you can.
</details>

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc (hunk 9 above)
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:109]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'Therefore' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:109]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'be loaded' may be passive voice. Use active voice if you can.

File: docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc [hunk 10]
```diff
@@ -124,13 +119,10 @@ The implementation of the function works analogously to the `registerIncluded` m
 You can then implement the `registerExcluded()` method at any point in the **boot function** of your service provider. The `registerExcluded()` takes an array of the names of the query parameters you want to exclude as its parameter. Here, only the query parameter "source" is excluded from caching: +
```
Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:119]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.TooWordy] 'implement' is too wordy.

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:119]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Weasel] 'only' is a weasel word!

Comment (plenty-cicd-3) [inline docs/en-gb/modules/plentyshop-plugins/pages/content-caching.adoc:119]: ⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[write-good.Passive] 'is excluded' may be passive voice. Use active voice if you can.

**Review by Tim-M-S — APPROVED** [review]
